docs/index.bs

```diff
@@ -3707,6 +3986,28 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Return true.
       1. Return false.
   </section>
+
+  <section algorithm>
+    <h3 id="lookup-race-response-algorithm"><dfn export>Lookup Race Response</dfn></h3>
+
+      : Input
+      :: |request|, a [=/request=]
+      : Output
+      :: a [=/response=] or null
+
+      1. If |request|'s [=request/reserved client=] is null, return null.
+      1. Let |storage key| be the result of running [=obtain a storage key=] given |request|'s [=request/reserved client=].
+      1. Let |registration| be the result of running <a>Match Service Worker Registration</a> given |storage key| and |url|.
```

Comment:
@sisidovski 
|url| looks not defined yet?
I suppose this is |request|'s [=request/url=].
Would you mind if I fix this?

Reply:
You're right. In the previous patch we used to have |url| as an input. Let me check previous commit just in case, and I'll commit the fix.

Reply:
Fixed it.

This is the previous commit, fyi.
https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10/commits/f8a0d3df2f4037d1d067cf52a0a67f7759949d3a#diff-7bdebf23eab87aaf2676a41586076394031b6b98fde24e35d1672aa65a1cc0feR3938

Reply:
thanks.